kitchen/bin/container-run.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
```

Comment:
A one line description and usage example would be nice in the script as comments.